src/data/seeds/000-cleanup.js

```diff
@@ -0,0 +1,3 @@
+const cleaner = require('knex-cleaner');
+
+exports.seed = knex => cleaner.clean(knex);
```

Comment:
I'm not familiar w/ knex-cleaner. I just looked it up, you'll have to explain it to me more tomorrow.